guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,435 @@
+
+### Quality Assurance {#quality-assurance}
+
+#### Metadata Quality Assurance {#metadata-quality-assurance}
+
+While studies are routed through the hospital DICOM routing infrastructure, images are validated with classification algorithms. Validating the usability of an image or data. Does the metadata associated with an image match the image? For instance, is this a left or right femur? The label states one direction but the image suggests another. These workloads allow for image corrections or create implied routing paths for further AI processing.
+
+**Clinical Workflow**
+
+1. Studies get routed in a large hospital DICOM routing system, at peak hour, the hospital is seeing about 300 studies per hour.
+2. As series arrive to DICOM router, DICOM router sends selected slices out of the series an AI processing system
+3. The AI processing system selects the correct slice or two for classification based on DICOM metadata.
+4. The AI algorithm evaluates the image metadata with the pixels and determines if the metadata (i.e. left shoulder) is indeed what the pixel data is showing (classification for left shoulder from image data).
+5. After classifying the image the AI system sends a response back to the appropriate DICOM target location.
+6. After receiving the response the DICOM router can route the image accordingly, either to the rejected queue for further analysis by QA technicians, or if everything looks good to the radiology work queue.
+
+**Clinical Workflow (without DICOM Router)**
+
+1. Studies are initiated on a modality and the DICOM is sent directly to a generalized application that ingests raw DICOM files.
+2. The application will read the DICOM metadata and select a single image or slice from the series and send a significantly reduced dataset to the AI platform and algorithm specific for the QA study associated with the dataset. Note: in this example the reduction is in some aspect of the overall data set to simplify the AI processing complexity in proportion to the algorithm accuracy.   In some variations, the application will send several image metadata and pixel sets in parallel to an AI platform and parse the results collectively.
+3. The AI platform will evaluate the image metadata and pixels to validate the pair. In this example a classification solution is used and the results are sent back to the application.
+4. After classification results are sent back to the application it adds metadata to the DICOM images that can be used later to filter the correct series of images for radiology studies. These DICOMs are added to the PACs along with the original data and made available to QA technicians.
```

Comment:
Hyehoon
On the 53rd line, it is just a typo. "PACs" should be revised to "PACS"